geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala

```diff
@@ -40,6 +41,8 @@ class RasterStore(val rasterOps: RasterOperations) {
   def getRasters(rasterQuery: RasterQuery): Iterator[Raster] = rasterOps.getRasters(rasterQuery)
 
   def putRaster(raster: Raster) = rasterOps.putRaster(raster)
+
+  def getBounds() = rasterOps.getBounds()
```

Comment:
If this is the 'interface' trait, I'd suggest type annotating public functions like this.